packages/eslint-plugin-template/docs/rules/interactive-supports-focus.md

```diff
@@ -555,8 +555,8 @@ The rule does not have any configuration options.
 <button (click)="onClick()" class="foo">Foo</button>
 <option (click)="onClick()" class="foo">Food</option>
 <select (click)="onClick()" class="foo"></select>
-<summary (click)="onClick()">Foo</summary>
-<textarea (keypress)="onKeypress()" class="foo"></textarea>
+<summary (click)="onClick()" aria-expanded>Foo</summary>
```

Comment:
[Deque does recommend using `aria-expanded=true|false` with the `<summary>` element](https://dequeuniversity.com/library/aria/expand-collapse-summary) because not all browsers support inferring the state from the parent `<details>` `open` attribute. However, as a lint rule, I don't think the attribute should be required with semantic HTML since it is only a recommendation.